src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carRental = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  const totalPrice = days * carRental;
+  const totalPriceSmallDiscount = days * carRental - smallDiscount;
+  const totalPriceBigDiscount = days * carRental - bigDiscount;
+
+  if (days < 3) {
+    return totalPrice;
+  }
+
+  if (days < 7) {
+    return totalPriceSmallDiscount;
+  }
+
+  return totalPriceBigDiscount;
 }
```

Comment:
these 2 variables are abundant

```suggestion

  if (days < 3) {
    return totalPrice;
  }

  if (days < 7) {
    return days * carRental - smallDiscount;
  }

  return days * carRental - bigDiscount;
}
```